manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: GadgetsRevived.GadgetsRevived
+PackageVersion: 1.0

```

```suggestion
PackageVersion: 2.0
```
Should match control panel. Version folder will need to be updated when changed